test/e2e/scenario/agents.scenario.ts

```diff
@@ -94,19 +94,15 @@ describe('rum', () => {
   })
 
   it('should not track disallowed cross origin xhr timings', async () => {
+    if (browser.capabilities.browserName === 'MicrosoftEdge') {
+      pending('Edge 18 seems to have valid timings even on cross origin requests')
```

Comment:
This is unfortunate. Anyway, this comment isn't really relevent. Maybe
```suggestion
      pending("Edge isn't always sending the timings")
```